public/sass/hof.scss

```diff
@@ -100,4 +105,66 @@
 .goldplated-label {
 	@extend .gold-label;
 	background-color: $goldplated-label;
-}
\ No newline at end of file
+}
+
+.stats-helper {
+	display: inline-block;
+	width: 30%;
+	vertical-align: top;
+}
+
+.hof-stats {
+	p {
+		font-size: 0.9rem;
+		margin-left: 1rem;
+		&:first-of-type {
+			margin-top: 0.5rem;
+		}
+	}
+	.regNum, .gpNum, .gNum {
+		position: relative;
+	}
+
+	.regNum:before {
+		content: "";
+		position: absolute;
+		width: 10px;
+		height: 10px;
+		background-color: $light-gray;
+		-moz-border-radius: 5px;
+		-webkit-border-radius: 5px;
+		border-radius: 5px;
+		left: -1rem;
+		top: 0.25rem;
+	}
+
+	.gpNum:before {
+		@extend .regNum:before;
+		background-color: $goldplated-label;
+	}
+
+	.gNum:before {
+		@extend .regNum:before;
+		background-color: $gold-label;
+	}
+}
+
+#chart {
+	margin-top: 0.25rem;
+	margin-left: -2rem;
+	display: inline-block;
+	g path:first-child {
```

Comment:
@nikilselvam Great question! The g element is used to group svg objects. One of the benefits of using SVGs to create vial elements is that they can be controlled with code, specifically CSS. More detailed info can be found here: https://developer.mozilla.org/en-US/docs/Web/SVG/Element/g